docs/filter_kgraph_discrete_kedge_attribute.md

```diff
@@ -0,0 +1,55 @@
+# filter kgraph (discrete kedge attribute)
+
+_subclass of [filter_kgraph](./filter_kgraph.md)_
+
+This operation removes kgraph edges which have a discrete attribute containing the specified value. Edges without the given attribute are left alone.
+
+### examples
+
+- [input](../examples/fill_and_filter/messages/11_filtered_kgraph_discrete_attribute_input.json), [output](../examples/fill_and_filter/messages/12_filtered_kgraph_discrete_attribute_output.json)
+
+### input requirements
+
+None
+
+### output guarantees
+
+- no edges in the kgraph which have the specified attribute with the given attribute.
+
+### allowed changes
+
+- remove knodes
+- remove kedges
+
+### parameters
+
+```yaml
+properties:
+  edge_attribute:
+    description: The name of the edge attribute to filter on.
+    example: provided_by
+    type: string
+  qedge_keys:
+    description: This indicates if you only want to remove edges with specific edge_keys.
+      If not provided or empty, all edges will be filtered on.
+    example:
+    - e01
+    type: array
+  qnode_keys:
+    default: []
+    description: This indicates if you only want nodes corresponding to a specific
+      list of qnode_keys to be removed. If not provided or empty, no nodes will be
+      removed when filtering. Allows us to know what to do with the nodes connected
+      to edges that are removed
+    example:
+    - n01
+    type: array
+  remove_value:
+    description: The value for which all edges containing this value in the specified
+      edge_attribute should be removed.
+    example: infores:semmeddb
```

Comment:
This attribute example has `infores:` prefix, yet the other filters don’t (eg. `normalized_google_distance`). Should be more specific about where exactly this operation is looking; `attribute_type_id`, `original_attribute_name` etc.)

Reply:
So in the ARAX implementation we looks in both attribute_type_id and original_attribute_name. Should we specify that it does that here as well?

Reply:
Also, this is a part where it changes how it works depending on the TRAPI version. In the old version of TRAPI which all of our examples thus far have been done in it would look in name. But name no longer exists in the current version of TRAPI.

Reply:
@finnagin While below it has been changed to infores:rtx-kg2, the example value has not been updated. Was this intentional?